_drafts/SC3-3-2-form-field-has-name.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Form field has accessible name
+description: |
+  Each form field element has an accessible name
+
+success_criterion:
+- 3.3.2 # Labels or Instructions
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rules applies to:
+- any element that is or [exposed to assistive technologies](#exposed-to-assistive-technologies),
```

Comment:
Depending on how that definition shakes out - should it include focusable elements as well? What about visible on the page?

Reply:
@WilcoFiers we propose that focusable elements be covered in the definition for 'exposed to assistive technologies'
We also don't think that the applicability should include elements that are simply 'visible on the page' on the premise that this rule is only concerned with ensuring AT form controls are labeled.